cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/AbstractChunkedLogProducer.java

```diff
@@ -35,7 +35,7 @@
 public abstract class AbstractChunkedLogProducer extends BodyProducer {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractChunkedLogProducer.class);
 
-  protected static final int BUFFER_BYTES = 8192;
+  static final int BUFFER_BYTES = 8192;
```

Comment:
did you intend to make it private ? or is it used in same package ? add comment if that's the case. should it rather be in constants ?

Reply:
Its used in derived classes. What do we need the comments for?

Reply:
I agree that this should remain protected, as there is no reason to open this up to be package private.